be/src/vec/functions/function_ip.h

```diff
@@ -969,4 +930,114 @@ class FunctionIPv6CIDRToRange : public IFunction {
     }
 };
 
-} // namespace doris::vectorized
\ No newline at end of file
+class FunctionIsIPv4Compat : public IFunction {
+public:
+    static constexpr auto name = "is_ipv4_compat";
+    static FunctionPtr create() { return std::make_shared<FunctionIsIPv4Compat>(); }
+
+    String get_name() const override { return name; }
+
+    size_t get_number_of_arguments() const override { return 1; }
+
+    DataTypePtr get_return_type_impl(const DataTypes& arguments) const override {
+        return make_nullable(std::make_shared<DataTypeUInt8>());
+    }
+
+    bool use_default_implementation_for_nulls() const override { return true; }
```

Comment:
no need override the function， the default value is `true`

Reply:
okay, i fogot to delete, i will delete it